index.html

```diff
@@ -3,116 +3,146 @@
   <head>
     <meta charset="utf-8">
     <title>Startrly</title>
+    <link href="css/normalize.css" rel="stylesheet">
+    <link href="css/style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
   </head>
```

Comment:
Only limited changes to the HTML (classes added for selection): Check!  You added labels to make it easier to find things in the CSS sheet and you added wrapping elements to group stuff together.